metadata/metadata.go

```diff
@@ -145,7 +145,13 @@ func (m *MetadataClient) getContainersDnsRecords(dnsEntries map[string]utils.Dns
 				logrus.Errorf("Skipping container %s: Invalid IP address %s", container.Name, externalIP)
 			}
 
-			fqdn := utils.FqdnFromTemplate(config.NameTemplate, container.ServiceName, container.StackName,
+			var nameTemplate string
+			nameTemplate, ok := service.Labels["io.rancher.service.external_dns_name_template"]
```

Comment:
Probably it is just due to lack of knowledge of golang (these are first few lines I have written in Go).
After I have read your comment I understood that `:=` means short variable definition (https://tour.golang.org/basics/10), not just assignment.

Let me update PR with correct code.

Also, is there some CI for the project (Travis/smth else), so that my pull request could be validated by machine first?